spec/models/countries_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require_relative File.join(File.dirname(__FILE__), '..', '..', 'app', 'models', 'countries')
```

Comment:
You can replace these 2 lines with `require 'rails_helper'`.

`spec_helper` [is already required](https://github.com/guidance-guarantee-programme/output/blob/master/.rspec#L2), perhaps we should add `rails_helper` to `.rspec` and remove the require from all the spec files